contrib/lang/html/packages.el

```diff
@@ -96,3 +96,31 @@ which require an initialization must be listed explicitly in the list.")
 (defun html/init-slim-mode ()
   (use-package slim-mode
     :defer t))
+
+(evil-leader/set-key-for-mode 'web-mode
+  "mgp" 'web-mode-element-parent
+  "mgc" 'web-mode-element-child
+  "mgb" 'web-mode-element-beginning
+  "mgn" 'spacemacs/html-navigation-micro-state
+  "mgs" 'web-mode-element-sibling-next
+  ;; v like 'View'
+  "mvx" 'web-mode-dom-xpath
+  "mve" 'web-mode-dom-errors-show
+  ;; i like 'Item'
+  "mir" 'web-mode-element-rename
+  "miw" 'web-mode-element-wrap
+  "mic" 'web-mode-element-clone
```

Comment:
to be consistent with the other we can put it on `mrc` even if it's not a refactoring command.